### PR TITLE
Adding steps to deploy vscode online with Foam in different clouds.

### PR DESCRIPTION
**Pull request by Klaudioz**
- Added first cloud: DigitalOcean

### CODE REVIEW

File: docs/vscode-online.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+It is possible to host an instance of Visual Studio Code in the cloud and have an URL with all your files (protectd with a password), being able to use it in any browser and device.
+
+## Foam with DigitalOcean
+
+- Requirements: a DigitalOcean account and Foam hosted in git (GitHub, Azure DevOps, etc). Extra: Needed a domain and a certificate to have a url like: https://foam.yourdomain.com/
+
+### Create the droplet
+- Go to: Droplets, push button: "Create Droplet" and over Choose and image go to marketplace and select: "code-server"
+- Next choose plan: Basic, 10/mo (I think the 5/mo plan can be not enough)
```
Comment (riccardoferretti) [inline docs/vscode-online.md:11]:
```suggestion
- Next choose plan: Basic, 10/mo (I think the 5/mo plan might not be enough)
```

File: docs/vscode-online.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# VSCode online
+
+It is possible to host an instance of Visual Studio Code in the cloud and have an URL with all your files (protectd with a password), being able to use it in any browser and device.
```
Comment (riccardoferretti) [inline docs/vscode-online.md:3]:
```suggestion
It is possible to host an instance of Visual Studio Code in the cloud and have an URL with all your files (protected with a password), being able to use it in any browser and device.
```

File: docs/vscode-online.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Foam with DigitalOcean
+
+- Requirements: a DigitalOcean account and Foam hosted in git (GitHub, Azure DevOps, etc). Extra: Needed a domain and a certificate to have a url like: https://foam.yourdomain.com/
+
+### Create the droplet
+- Go to: Droplets, push button: "Create Droplet" and over Choose and image go to marketplace and select: "code-server"
```
Comment (riccardoferretti) [inline docs/vscode-online.md:10]: I am not sure I understand this sequence, can you plz rephrase it or break it down further?

File: docs/vscode-online.md (hunk 1 above)
Comment (Klaudioz) [reply docs/vscode-online.md:11]: Thanks for your comment. I'm using the 5/mo version right now and it looks pretty good.

File: docs/vscode-online.md (hunk 3 above)
Comment (Klaudioz) [reply docs/vscode-online.md:11]: I tried to explain it better. Please let me know if it's ok for you.

Comment (riccardoferretti) [reply docs/vscode-online.md:11]: The updates on the rest of the guide are really good!
I am still a bit confused about this specific line :) what do you mean by "over Choose and image"? Is it `hover "Choose an image"`? then you go to a "marketplace" label/button?
Maybe you can add quotes around labels to highlight them from the instruction

Comment (Klaudioz) [reply docs/vscode-online.md:11]: It'd be an instead and !! 😂
Fixing it now.

**Comment by Klaudioz** [comment]
Please don't push this guide yet because there are issues with the version which DigitalOcean is installing in the Droplet. It looks very old so using the trick to provide the VSIX and install from VSIX is not working. 

Unable to install extension 'foam.foam-vscode' as it is not compatible with VS Code '1.43.2'. The one I have in my kubernetes cluster is more updated and it worked: VS Code: v1.45.1.

Anybody can provide me Foam v0.2.0.vsix file ?

**Comment by riccardoferretti** [comment]
cc @jevakallio in case he can help, I am not familiar with this

**Comment by Klaudioz** [comment]
If with an older version of the Foam's vsix still not working I'll need to redo the guide because DigitalOcean is providing a very old version of vscode and I'm unable to update it so it'd work if I don't use its automated process.

**Comment by Klaudioz** [comment]
No way to do it in an easy way, unfortunately :(

**Pull request closed without merging**